### ISSUE TITLE
Model

### ISSUE THREAD

**Issue by vtqtuan** [zengxianyu/jsws#16]
Can you provide trained models for us to test demo pictures?

**Comment by zengxianyu** [comment]
No. The trained model file was lost. You may train a model using the provided training script. I might re-train a model later and upload a new trained model file
